src/hooks-log.coffee

```diff
@@ -0,0 +1,23 @@
+util = require 'util'
+
+hooksLog = (logs = [], logger, logVariant, content) ->
+  if arguments.length is 4 and logVariant in ['info', 'debug', 'warn', 'verbose', 'error', 'log']
```

Comment:
Can you please articulate your motivation for this? I think that it's never used and it's only making the interface more confusing.